### ISSUE TITLE
Doing a .get on a queryset adds deleted objects

### ISSUE THREAD

**Issue by skorokithakis** [makinacorpus/django-safedelete#131]
I have a model that contains a deleted task (`Butter knife`)

```
<SafeDeleteQueryset [<Task: MRI>, <Task: LINAC>, <Task: GammaKnife>, <Task: CyberKnife>, <Task: Banana stand>, <Task: Butter knife>]>

>>> g
<Task: MRI>

>>> tasks = Task.objects.filter(department=d)
>>> tasks.count()
5
>>> tasks = Task.objects.filter(department=d)
>>> tasks.get(pk=g.id)
<Task: MRI>
>>> tasks.count()
6
```

I'm not getting a deleted task, I'm just getting a random, non-deleted task, yet that queryset then contains the deleted task as well.

**Comment by AndreasBackx** [comment]
What object is deleted and when is it deleted? We need some more information before we can help you.

**Comment by Gagaro** [comment]
The issue is in our `get` method modifying the instance of the queryset. I'll try to fix/release that tomorrow.

**Comment by Gagaro** [comment]
Waiting for travis before releasing it.

**Comment by Gagaro** [comment]
released in 0.5.3
